### PR TITLE
js choropleth mapbox

### PR DESCRIPTION
**Pull request by Mahdis-z**

### CODE REVIEW

File: _posts/plotly_js/maps/choropleth-mapbox/2019-08-16-choropleth_plotly_js_index.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: JavaScript Graphing Library D3.js-based Choropleth Mapbox | Examples | Plotly
+name: Choropleth Mapbox
+permalink: javascript/choropleth-mapbox/
+description: How to make a D3.js-based choropleth mapbox in JavaScript. A Choropleth mapbox shades geographic regions by value.
+layout: user-guide
+thumbnail: thumbnail/choropleth.jpg
```
Comment (michaelbabyn) [inline _posts/plotly_js/maps/choropleth-mapbox/2019-08-16-choropleth_plotly_js_index.html]: We should change this thumbnail so it uses [the same one as in the python docs](https://github.com/plotly/plotly.py-docs/blame/991278447671118a5cd26ee1e18a13c21451260c/python/mapbox-county-choropleth.md#L36).

File: _posts/plotly_js/maps/choropleth-mapbox/2019-08-16-choropleth-geojson-object.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+name: Streets Tile, Needs Token
+plot_url: https://codepen.io/plotly/embed/NWKRygN/?height=510&theme-id=15263&default-tab=result
+language: plotly_js
+suite: choropleth-mapbox
+order: 2
+sitemap: false
+arrangement: horizontal
+markdown_content: |
+
+    The following example sets `geojson object` of type `feature` and geometries of type 'Polygone'. For more information see [geojson attribute](https://plot.ly/javascript/reference/#choroplethmapbox-geojson) in the reference page.
```
Comment (michaelbabyn) [inline _posts/plotly_js/maps/choropleth-mapbox/2019-08-16-choropleth-geojson-object.html]: Typo in 'Polygon'.

```suggestion
    The following example sets `geojson object` of type `feature` and geometries of type 'Polygon'. For more information see [geojson attribute](https://plot.ly/javascript/reference/#choroplethmapbox-geojson) in the reference page.
```

File: _posts/plotly_js/maps/choropleth-mapbox/2019-08-16-basic-choropleth1.html [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+var data = [{
+  type: "choroplethmapbox", locations: ["NY", "MA", "VT"], z: [-50, -10, -20],
+  geojson: "https://raw.githubusercontent.com/python-visualization/folium/master/examples/data/us-states.json"
+}];
+
+var layout = {mapbox: {center: {lon: -74, lat: 43}, zoom: 3.5},
+              width: 600, height:400};
+
+var config = {mapboxAccessToken: "pk.eyJ1IjoiZXRwaW5hcmQiLCJhIjoiY2luMHIzdHE0MGFxNXVubTRxczZ2YmUxaCJ9.hwWZful0U2CQxit4ItNsiQ"};
```
Comment (michaelbabyn) [inline _posts/plotly_js/maps/choropleth-mapbox/2019-08-16-basic-choropleth1.html]: If we could move these tokens out of the main text, that would be good. We can replace them with something like "add-your-mapbox-token-here"

File: _posts/plotly_js/maps/choropleth-mapbox/2019-08-16-US_states.html [hunk 4]
```diff
@@ -0,0 +1,23 @@
+---
+name: Dark tile, Needs Token
+plot_url: https://codepen.io/plotly/embed/eYOdVRG/?height=527&theme-id=15263&default-tab=result
+language: plotly_js
+suite: choropleth-mapbox
+order: 3
+sitemap: false
+arrangement: horizontal
+markdown_content: |
+    This example uses [zmin and zmax](https://plot.ly/javascript/reference/#choroplethmapbox-zmin) to define the lower bound and upper bound of the color domain.
```
Comment (michaelbabyn) [inline _posts/plotly_js/maps/choropleth-mapbox/2019-08-16-US_states.html]: Can we add "If these attributes are not set, Plotly will define the bounds of the color domain based on the input data."?

Comment (Mahdis-z) [reply _posts/plotly_js/maps/choropleth-mapbox/2019-08-16-US_states.html]: Yeah, as `zauto` is true by default.

File: _posts/plotly_js/maps/choropleth-mapbox/2019-08-16-US_states.html [hunk 5]
```diff
@@ -7,7 +7,7 @@
 sitemap: false
 arrangement: horizontal
 markdown_content: |
-    This example uses [zmin and zmax](https://plot.ly/javascript/reference/#choroplethmapbox-zmin) to define the lower bound and upper bound of the color domain.
+    This example uses [zmin and zmax](https://plot.ly/javascript/reference/#choroplethmapbox-zmin) to define the lower bound and upper bound of the color domain. If these attributes are not set, Plotly [determines the color domain](https://plot.ly/javascript/reference/#heatmap-zauto) based on the input data.
```
Comment (michaelbabyn) [inline _posts/plotly_js/maps/choropleth-mapbox/2019-08-16-US_states.html]: :+1:

**Pull request merged**